web_pivot_computed_measure/static/src/js/pivot_model.js

```diff
@@ -102,29 +103,78 @@ odoo.define('web_pivot_computed_measure.PivotModel', function (require) {
             return _.contains(this.data.measures, field);
         },
 
+        /**
+         * Helper function to add computed measure fields data into 'dataPoint'
+         *
+         * @param {Object} dataPoint
+         * @param {Object} dataPointComp
+         */
+        _fillComputedMeasuresData: function (dataPoint, dataPointComp) {
+            for (var cm of this._computed_measures) {
```

Comment:
This syntax is not available in the ES version supported in Odoo 12: https://travis-ci.org/github/OCA/web/jobs/670341645#L1434